<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pt.isel.ls.utils.TryCatchTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2024-03-27T14:29:48" hostname="LAPTOP-RPUNL8UP" time="0.008">
  <properties/>
  <testcase name="test of tryCatch with NoSuchElementException()" classname="pt.isel.ls.utils.TryCatchTest" time="0.001"/>
  <testcase name="test of tryCatch with IllegalStateException()" classname="pt.isel.ls.utils.TryCatchTest" time="0.001"/>
  <testcase name="test of tryCatch with IllegalArgumentException()" classname="pt.isel.ls.utils.TryCatchTest" time="0.002"/>
  <testcase name="test of tryCatch with exception message()" classname="pt.isel.ls.utils.TryCatchTest" time="0.001"/>
  <testcase name="test of tryCatch without an exception()" classname="pt.isel.ls.utils.TryCatchTest" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
